src/videojs-contrib-hls.js

```diff
@@ -677,6 +679,8 @@ videojs.HlsSourceHandler = HlsSourceHandler;
 videojs.Hls = Hls;
 videojs.m3u8 = m3u8;
 videojs.registerComponent('Hls', Hls);
+videojs.plugin('gapSkipper', gapSkipper);
```

Comment:
It's been decided that this should not be exposed as a plugin since it is not going to be used by any other projects soon.